### PR TITLE
Add GitHub Action docs

### PR DESCRIPTION
**Pull request by wbeuil**
For issue cloudskiff/driftctl/issues/281

### CODE REVIEW

File: docs/ci_cd/guides/ghaction.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+jobs:
+  scheduled:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: us-east-1
+    steps:
+      - name: Checkout
```
[the remark this answers is not in the document]
Comment (wbeuil) [reply docs/ci_cd/guides/ghaction.mdx:28]: Naming each steps is not required at all

File: docs/ci_cd/guides/ghaction.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        uses: actions/checkout@v2
+      - name: Run driftctl
+        uses: cloudskiff/driftctl-action@v1
+        env:
+          DCTL_NO_VERSION_CHECK: true
```
[the remark this answers is not in the document]
Comment (wbeuil) [reply docs/ci_cd/guides/ghaction.mdx:33]: This was to show how to use env variable like `DCTL_NO_VERSION_CHECK` but I can remove it if you want

File: docs/ci_cd/guides/ghaction.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (eliecharra) [reply docs/ci_cd/guides/ghaction.mdx:28]: I mean there is no need to checkout the project IMHO

File: docs/ci_cd/guides/ghaction.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (eliecharra) [reply docs/ci_cd/guides/ghaction.mdx:33]: Yeah I think we should remove it. We should push our CI users to use latest version. You should have very particular reason to use a specific version.

**Review by eliecharra — APPROVED** [review]

**Pull request merged**